cmd/draft/installer/install.go

```diff
@@ -92,7 +93,11 @@ spec:
     spec:
       containers:
       - name: draftd
-        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+      {{- if .Values.imageOverride }}
```

Comment:
This could be simplified to `{{ default .Values.imageOverride (printf "%s:%s" .Values.image.repository .Values.image.tag) }}`